### DISCUSSION TITLE
生成式AI是否會使人們蒐集資料的能力減弱

### DISCUSSION THREAD

**Discussion by aswyz4260** [1101-GS4719-C/Python-Community-GS4719-C#1998] [General]
其實一直在思考一個問題，生成式AI或聊天機器人這類型的技術，會不會減弱人們蒐集資料的能力？大家不再需要用精煉的關鍵字去搜尋各類資料，AI需要的反而是人們完整且具體的描述，甚至隨著語言模型的進步，或許不那麼完整的語句也可以獲得需要的解答。

然而，又想到一個問題，如果AI發展為必然的趨勢，那麼以前那種關鍵字搜尋的方式定會走入歷史，如此一來，前段所述還是「問題」嗎？

但是如何正確問問題、準確切入要點，應該是人們無論是面對AI或是真實人類（？）都應該學習的能力對吧？那麼生成式AI對於人類的語言能力是助益還是會有所減損呢？

想聽聽大家的看法。

**Comment by JustAHedgehog** [comment]
我認為 AI 確實改變人類的搜尋資料或撰寫文本的行為，但我覺得並不會導致一班的搜尋引擎消失，因為目前生成式 AI 仍無法提供 100% 正確的答案以及此答案的依據，像我在查詢資料時我則發現chatgpt所得出的結論來源無法查詢，所以如果是要做正式文件的參照仍需自行去搜尋引擎查詢比較可靠。
最後我認為在搜尋方面，以後生成式 AI 將變成輔助引導新手搜尋文件，像我們一開始學習程式遇到報錯時不也是胡亂複製貼上嗎？也是在別人的教導下才漸漸知道究竟要搜尋甚麼才會找到我們要的資訊，而未來 AI 就將扮演那位願意教導我們的「別人」。

**Comment by alexender0104** [comment]
我覺得生成式AI確實可以讓人們更輕鬆地獲得所需的資料或內容，但這並不一定會導致人們的資料蒐集能力減弱。實際上，生成式AI可能會激勵人們更多地去尋找、理解和處理資訊，因為這些技術僅是工具，而非終極解答。人們仍然需要對所獲得的資訊進行評估、分析和應用，而這些都需要扎實的知識和技能。

**Comment by hbh10601** [comment]
這幾天GPT爆掉真的對我來說造成很大的不便，不過我覺得不會減弱能力(畢竟他有時候給的答案是錯的)，而是減低蒐集資料的動力，有種被寵壞的感覺

**Comment by tshsu04224** [comment]
我覺得生成式AI的出現大量減少了我們日常生活中到處找資料、查google等原先使用資源的利用，但我們縮短了許多花費在查詢資料的時間，也增加了我們得到資源的效率，而且要準確的得到我們想要的答案，必須先精準並詳細的闡述自己的問題或要求，AI才會給你響應完整的答覆，因為我認為生成式AI的出現或許能增進我們找到答案的邏輯能力

**Comment by jeff110303549** [comment]
我認為生成式AI是提升了信息搜集的效率和便捷性，使得使用者不需花費大量時間篩選相關資料即可獲得所需的資訊。AI能夠處理複雜的自然語言查找，簡化信息搜集過程，通過這種快速提供相關信息的能力，AI幫助使用者節省大量的時間，使他们能更專注於具有創造性的工作抑或决策，提高整體生產力和知識獲取的效率。

**Comment by sandyho0307** [comment]
我認為無論是面對AI還是面對真實的人類，正確地提出問題和準確切入要點的能力仍然是非常重要的。這不僅僅是為了獲取資訊，更是為了培養我們的思維能力和表達能力，也就是說問對問題能讓我們更清晰地了解自己的需求和思考過程。

**Comment by yubo0oo** [comment]
我們不再需要透過精煉的關鍵字去搜尋資料，而是可以直接以自然的語言描述來獲得需要的解答。這樣的技術可能會降低人們整理、分析資料的能力，但同時也提供了更便利、更直觀的資訊獲取方式。
至於生成式AI對於人類的語言能力是否有助益還是會有所減損，這取決於使用者的使用方式。如果人們僅僅依賴AI來溝通和獲取資訊，可能會導致語言能力的減損；但如果能夠適當地利用生成式AI來輔助學習和溝通，則反而可以提升語言能力，因為我們需要學習如何清晰地表達自己的想法，並學會如何正確地提問。

**Comment by 710388** [comment]
不會吧ai還是有很多漏洞自己上網搜尋或找到的資料還是比較詳盡一點

**Comment by 155Julia** [comment]
我認為生程式Ai確實會降低使用者的搜集資料能力，所以我認為在上大學前的學生是盡量不要太依賴Ai，培養好足夠的自我排查問題、解決問題、搜集資料等基本邏輯思判能力後，在大量使用會比較好。

**Comment by Piyenparty** [comment]
我其實覺得會，生成式 AI 的確有可能讓使用者在資料搜集這方面變得比較依賴，甚至會影響到他們主動解決問題的能力。尤其是對還沒進入大學的學生來說，應該要先把基本的邏輯思考、問題排查和資料查找能力建立起來，再來大量使用 AI 會比較適當。這樣可以確保他們是真的具備獨立思考的能力，而不是完全依賴工具來給答案。

**Comment by wuyu06** [comment]
我覺得會 但這也不見得是一件壞事 生成式AI確實可能讓我們不再像以前那樣花時間用關鍵字蒐集資料，但同時也要求我們更會用語言去描述問題，這是另一種思考能力的轉變。或許未來重點不在怎麼搜尋，而是怎麼提問和判斷AI回覆的準確度

**Comment by edawdewv** [comment]
AI 確實改變了我們搜尋和撰寫資料的方式，但目前生成式 AI 還無法完全取代傳統搜尋引擎，特別是在資訊的正確性和來源可查性上還存在限制。若要用於正式文件或研究，我認為仍需自行確認資料來源比較可靠。目前 AI 更像是一位「指引者」或「助教」，幫助初學者更快找到所需資訊，就像我們剛學程式時靠別人的指導一樣。這種輔助功能反而能提升學習效率和效果。

**Comment by Pyra-0** [comment]
我覺得不可避免，資料蒐集能力一定會變差。現在的AI大多數都能在你問出問題的下一刻開始生成你需要的資料，過程直接了當，這會變成人們對其產生依賴，導致資料蒐集能力變差，但現階段的AI，回答出來的東西不一定是對的，有時又卻能引出人對於資料判斷的能力。對於AI，我認為不應該甚麼都問它，而是先自己思考過後，真的不行在尋求協助，像是問助教或教授那樣，避免喪失基本的邏輯批判和資料查找能力。

**Comment by yuku-huang** [comment]
生成式AI確實讓人少了自己動手查資料的動力，但也逼你更會「問問題」。它不是削弱能力，而是轉移焦點：從「怎麼找資料」變成「怎麼問對問題」。關鍵字搜尋會退場沒錯，但邏輯清楚、描述精準的能力反而變得更重要。資料蒐集不是死背資料來源，而是懂得整理、判斷、轉化。這部分AI幫不了你，只能輔助。所以會不會退化，關鍵在於你怎麼用它。

**Comment by yuku-huang** [comment]
會。會寫程式就像會寫字、會數學一樣，是種基本素養。AI 可以幫你寫，但你得看得懂、改得對、用得巧。不會寫程式的人，用 AI 只是猜；會寫的人，才能指揮 AI 幫你飛。未來不是不需要程式人，是只留下會用 AI 寫程式的人。
